randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -1412,8 +1412,10 @@ Extra - asset_id: collision_camera_019
   > Tunnel to Energy Recharge Station Secret
       All of the following:
           Morph Ball and After Ferenia - Recharge Station Secret Grapple Block
-          # Destroy bomb blocks
-          Bomb or Cross Bomb or Power Bombs ≥ 2
+          Any of the following:
+              # Destroy bomb blocks
+              Bomb or Cross Bomb
+              Power Bombs ≥ 2 and Lay Power Bomb
```

Comment:
can you rebase, and modify the changelog of the previous change to inlcude this path too?